app/components/card_component.html.erb

```diff
@@ -1,9 +1,12 @@
-<div <%== attributes(classes: card_classes) %>>
-  <%= content %>
+<div class="<%== card_classes_wrapper %>">
+  <% unless content.blank? %>
```

Comment:
Can this use [`if content?`](https://viewcomponent.org/api.html#content--boolean)?

Reply:
Ok I remember now. I was having some trouble with `content?` actually working right? I didn't investigate at the time. But check this out. I threw a _debugger_ in the template and when I print out the value of `content?` and `content` respectively I get _a Proc_ and _nil._ 

![image](https://github.com/zinc-collective/convene/assets/5185/ffc29b40-c6fa-4805-a735-9758f0610cb9)

Whereas if I print out `footer?` the value is `true`

Reply:
Ohhh I see what's happening. `content?` is technically truthy and returning a proc, because that proc is actually the nested `CardComponent` block in the `ManagementComponent`.

Reply:
That is weird as heck. I wonder if we're doing anything (like, defining a `content` or `content?` method). I found [the implementation](https://github.com/ViewComponent/view_component/blob/3487283035bbcb868e46f93fcfb6abadfeed650a/lib/view_component/base.rb#L269-L276) but it didn't shine much light for me.

Reply:
My goal was to avoid this empty white space above the Manage Marketplace navigation when there's no content. 
<img width="719" alt="image" src="https://github.com/zinc-collective/convene/assets/5185/0248f5b2-be1b-4ab3-9d57-df65ce7e3219">